src/test/java/racing/RacingTest.java

```diff
@@ -0,0 +1,64 @@
+package racing;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class RacingTest {
+    private List<Car> cars;
+    private Racing racing;
+
+    @BeforeEach
+    void set() {
+        cars = new ArrayList<>();
+        cars.add(new Car("chloe"));
+        cars.add(new Car("tt"));
+        racing = new Racing(cars);
```

Comment:
```suggestion
        racing = new Racing("chloe,tt", 2);
```
오직 테스트를 위해 존재했던 `public Racing(List<Car> cars)`생성자를 제거할 수 있을 것 같습니다.